### PR TITLE
Swift3

### PR DESCRIPTION
**Pull request by alexpersian**
- Initial revisions to prepare the style guide for Swift 3

### CODE REVIEW

**Comment by Benuuu** [comment]
:shipit:

**Comment by Benuuu** [comment]
If the compiler complains about any of these, we shouldn't include them in our styleguide.

e.g. dot notation for enums.

**Comment by giuggioRapido** [comment]
Currently in Swift 2 I’d write 
`func setChicken(chicken: Chicken)` and the caller would see `setChicken(aChicken)`

Since first parameters are given an external name by default, which is preferred:

`func setChicken(_ chicken: Chicken)` -> `setChicken(aChicken)`
v.s.
`func set(chicken: Chicken)` -> `set(chicken: aChicken)`

**Comment by alexpersian** [comment]
@giuggioRapido the second would be preferred. Using `_` should be avoided now unless absolutely necessary. There is more information regarding this in the [Swift API Design Guidelines](https://swift.org/documentation/api-design-guidelines/#argument-labels).

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ This is the documentation of Intrepid best practices and style regarding the Swi
 
 ##Making / Requesting Changes
 
-Feedback and changes are encouraged!  The current maintainers of this style guide are @brightredchili and @loganwright.  If you have an issue with any of the existing rules and would like to see something changed, please see the [contribution guidelines](CONTRIBUTING.md),
+Feedback and change are encouraged!  The current maintainers of this style guide are @brightredchili.  If you have an issue with any of the existing rules and would like to see something changed, please see the [contribution guidelines](CONTRIBUTING.md),
```
Comment (brightredchilli) [inline README.md]: I'm clearly not the only one mantaining it - while you're making this change, can you please change to 'developers of Intrepid'? ❤️

File: README.md [hunk 2]
```diff
@@ -440,6 +470,24 @@ func moveView(view: UIView, toFrame frame: CGRect)
...
+With Swift 3, first parameter labels are now considered default. This means that you will no longer need to worry about doubling up the parameter name if you want it to be exposed to callers.
+
+**Swift 3**
+
+```Swift
+func foo(x: Int, y: Int)
+// Callers will see foo(x: Int, y: Int)
+```
+
+**Swift 2**
```
Comment (brightredchilli) [inline README.md]: maybe add 'don't do'?

**Review by brightredchilli — COMMENTED** [review]
:sheep:

File: README.md (hunk 1 above)
Comment (alexpersian) [reply README.md]: Thanks, changed!

File: README.md (hunk 2 above)
Comment (alexpersian) [reply README.md]: This was changed over to be more clear in the latest commit.

**Comment by paulrolfe** [comment]
:shipit: Thanks Alex!

**Pull request merged**